community/config/nodes/add_node.md

```diff
@@ -43,7 +43,7 @@ Don't forget to account for OS overhead and other software requirements on machi
 * **Daemon SFTP Port**: the port the Daemon sftp-server or standalone SFTP server should listen on.
 
 ## Install the Daemon
-At this point you'll need to have the Daemon installed on your machine. Check out the [documentation](/wings/installing.html)
+At this point you'll need to have the Daemon installed on your machine. Check out the [documentation](/wings/1.0/installing.html)
```

Comment:
leaving out the 1.0 is actually correct here, as it'll (or at least should) redirect to the latest version.

Reply:
> leaving out the 1.0 is actually correct here, as it'll (or at least should) redirect to the latest version.

You're right! Thanks for pointing out. :)